packages/wallet/backend/tests/rafiki/service.test.ts

```diff
@@ -0,0 +1,42 @@
+import { Bindings } from '@/app'
+import { env } from '@/config/env'
+import { createContainer } from '@/createContainer'
+import { RafikiService } from '@/rafiki/service'
+import { Container } from '@/shared/container'
+import { Knex } from 'knex'
+import { TestApp, createApp } from '../app'
+import { truncateTables } from '../tables'
+import { mockOutgoingPaymenteCreatedEvent } from '../mocks'
+
+describe('Rafiki Service', () => {
+  let bindings: Container<Bindings>
+  let knex: Knex
+  let rafikiService: RafikiService
+  let appContainer: TestApp
+
+  beforeAll(async (): Promise<void> => {
+    bindings = createContainer(env)
+    appContainer = await createApp(bindings)
+    knex = appContainer.knex
+    rafikiService = await bindings.resolve('rafikiService')
+  })
+
+  afterAll(async (): Promise<void> => {
+    appContainer.stop()
+    knex.destroy()
+  })
+
+  afterEach(async (): Promise<void> => {
+    await truncateTables(knex)
+  })
+
+  describe('OnWebHook', () => {
+    it('should throw an error if the paymentPointer is invalid', async () => {
+      const webHook = mockOutgoingPaymenteCreatedEvent({})
+
+      await expect(rafikiService.onWebHook(webHook)).rejects.toThrowError(
+        /Invalid payment pointer/
+      )
+    })
+  })
```

Comment:
@rico191013 @Tymmmy How can I disable the eslint and typescript checking here to test the unknow event type ?

Reply:
```typescript
      const webHook = mockOutgoingPaymenteCreatedEvent({type: "mock-event"})

     // service.ts(25, 3): The expected type comes from property 'type' which is declared here on type 
     // 'Partial<WebHook>'
```

Reply:
I didn't send the code above, because it's getting error. But I want to understand how to disable the eslint at this line.

Reply:
I wanted to see in the code how could I disable the line, but I couldn't get the normal code and errors we have in testnet because of local repository PR. But please try : 
<img width="642" alt="image" src="https://github.com/interledger/testnet/assets/117268143/abaa9c6a-0074-4f65-afc9-ad7f017c278c">